source/ref_changelog.rst

```diff
@@ -1,6 +1,57 @@
 Changelog
 =========
 
+Odahu 1.2.0, 23 June 2020
+--------------------------
+
+New Features:
+""""""""""""
+
+- Core:
+    * Knative updated to latest version that makes it possible to deploy model services to different node pools.
+    * Go dependencies was globally updated to migrate from GOPATH to go modules.
+
+- PostgreSQL backend:
+    Now you can use PostgreSQL as database backend when deploying ODAHU. You can find additional documentation :ref:`here <gen_database:Postgres>`.
+
+- ODAHU CLI:
+    * Add option `--ignore-if-exist` for entities creation.
+    * Add output format options (yaml, json, table, jsonpath).
```

Comment:
```suggestion
```
We just added that these options exist, but they were previously.
![Screenshot 2020-06-19 at 16 04 03](https://user-images.githubusercontent.com/49116723/85135353-a1c94b00-b246-11ea-8dca-020ad218cab4.png)

I only added to the cli output what is highlighted in orange.

Reply:
You are correct, fixed in 21819aa